src/git.rs

```diff
@@ -0,0 +1,24 @@
+use git2::{build::CheckoutBuilder, build::RepoBuilder, Repository as GitRepository,
+           RepositoryInitOptions};
+use quicli::prelude::*;
+use std::fs::remove_dir_all;
+use std::path::PathBuf;
+use Cli;
+
+pub fn create(project_dir: &PathBuf, args: Cli) -> Result<GitRepository> {
```

Comment:
why not take `&Path` and `&Cli`?

Reply:
also I'd call this `create_from_git`

Reply:
re Cli, i just didn't need to. but don't really care. the module is called git, so it reads `git::create` - i'd rather not have the redundancy